config/application.rb

```diff
@@ -15,6 +15,8 @@ class Application < Rails::Application
     config.cache_max_age = ENV['CACHE_MAX_AGE'] || 10.seconds
 
     config.middleware.use Rack::BounceFavicon
+    require 'bounce_browserconfig'
```

Comment:
It could be a possibility but `config/application.rb` is the only place in the codebase where we;re explicitly pulling middleware.